clients/java/src/main/java/io/camunda/zeebe/client/impl/ZeebeClientImpl.java

```diff
@@ -181,9 +183,8 @@ public static ManagedChannel buildChannel(final ZeebeClientConfiguration config)
 
     configureConnectionSecurity(config, channelBuilder);
     channelBuilder.keepAliveTime(config.getKeepAlive().toMillis(), TimeUnit.MILLISECONDS);
-    channelBuilder.userAgent("camunda-client-java/" + VersionUtil.getVersion());
+    channelBuilder.userAgent("zeebe-client-java/" + VersionUtil.getVersion());
```

Comment:
❓ Why did we change this for the Zeebe client originally? I think this needs to stay. Was it due to the renaming of the artifact?

Reply:
Yes, I changed it while changing the artifact name. Good catch 👍